src/openforms/forms/api/serializers/form_step.py

```diff
@@ -65,6 +92,11 @@ class FormStepSerializer(serializers.HyperlinkedModelSerializer):
         parent_lookup_kwargs={"form_uuid_or_slug": "form__uuid"},
         read_only=True,
     )
+    translations = ModelTranslationsSerializer(
+        source="form_definition",
+        required=False,
+        read_only=True,
```

Comment:
the `read_only` is confusing me, and from the tests it looks like it's being ignored anyway since the fields are used in create/updates.

Reply:
I completely missed the `model` of this serializer and thought this was `FormDefinition`! Your reasoning makes perfect sense.